app/assets/javascripts/discourse/app/templates/mobile/components/parent-category-row.hbs

```diff
@@ -0,0 +1,61 @@
+{{#unless isHidden}}
+  {{plugin-outlet name="category-list-above-each-category" connectorTagName="" tagName="" args=(hash category=category)}}
+  <div data-category-id={{category.id}} data-notification-level={{category.notificationLevelString}} style={{border-color category.color}} class="category-list-item category {{if isMuted "muted"}}">
+    <table class="topic-list">
+      <tbody>
+        <tr>
+          <th class="main-link">
+            {{category-title-link category=category}}
+          </th>
+        </tr>
+        {{#if category.description_excerpt}}
+          <tr class="category-description">
+            <td colspan="3">
+              {{html-safe category.description_excerpt}}
+            </td>
+          </tr>
+        {{/if}}
+        {{#unless isMuted}}
+          {{#if showTopics}}
+            {{#each category.topics as |t|}}
+              {{mobile-category-topic topic=t}}
+            {{/each}}
+          {{/if}}
+        {{/unless}}
+        {{#if category.isGrandParent}}
+          {{#each category.subcategories as |subcategory|}}
+            {{sub-category-row category=subcategory listType=listType}}
+          {{/each}}
+        {{else if category.subcategories}}
+          <tr class="subcategories-list">
+            <td>
+              <div class="subcategories">
+                {{#each category.subcategories as |subcategory|}}
+                  {{sub-category-item category=subcategory listType=listType}}
+                {{/each}}
+              </div>
+            </td>
+          </tr>
+        {{/if}}
+      </tbody>
+    </table>
+    <footer class="clearfix">
```

Comment:
can you remove clearfix here please ? Don't think it does anything

Reply:
It's breaking the existing UI if remove it.

Reply:
@awesomerobot I know you were looking for cases where removing clearfix was actually breaking something

Reply:
![Screenshot from 2020-07-30 22-16-05](https://user-images.githubusercontent.com/9372109/88950539-6b093a80-d2b2-11ea-9900-ec79bcddb391.png)
`/categories` page in mobile view.